### PR TITLE
Proposed IFRS-compliant chart of accounts

### PR DESCRIPTION
**Pull request by khrlzhfr**
Significant classes of transactions (SCOTs/accounts) based on latest IFRS
Main headings in all caps for better presentation in financial statements
4-digit code to distinguish different SCOTs with same names and to sort them in order of liquidity

### CODE REVIEW

File: erpnext/accounts/doctype/account/chart_of_accounts/verified/ifrs_chart_of_accounts.py [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+				"is_group": 1
+			},
+			_("2600 Cash and cash equivalents"): {
+				_("2610 Cash bag A"): {
+					"account_type": "Cash"
+				},
+				_("2611 Cash bag B"): {
+					"account_type": "Cash"
+				},
+				_("2620 Current account 00000000 - Good Bank"): {
```
Comment (nabinhait) [inline erpnext/accounts/doctype/account/chart_of_accounts/verified/ifrs_chart_of_accounts.py]: Why naming includes "Good", "Better" etc?

File: erpnext/accounts/doctype/account/chart_of_accounts/verified/ifrs_chart_of_accounts.py [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+					"account_type": "Bank"
+				},
+				_("2621 USD current account 00000000 - Better Bank"): {
+					"account_type": "Bank"
+				},
+				_("2630 Savings account 00000000 - Better Bank"): {
+					"account_type": "Bank"
+				},
+				_("2631 GBP savings account 00000000 - Best Bank"): {
```
Comment (nabinhait) [inline erpnext/accounts/doctype/account/chart_of_accounts/verified/ifrs_chart_of_accounts.py]: Why using any specific currency?

File: erpnext/accounts/doctype/account/chart_of_accounts/verified/ifrs_chart_of_accounts.py [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+				_("4603 Service warranties"): {},
+				_("4604 Volume discounts and returns"): {},
+				_("4605 Legal claim"): {},
+				_("4606 Contingent liability"): {},
+				"is_group": 1
+			},
+			_("4700 Deferred revenue"): {
+				"is_group": 1
+			},
+			_("4xxx Liabilities directly associated with assets classified as held for sale"): {},
```
Comment (nabinhait) [inline erpnext/accounts/doctype/account/chart_of_accounts/verified/ifrs_chart_of_accounts.py]: can it be short?

**Comment by nabinhait** [comment]
Closing this PR as per https://github.com/frappe/erpnext/wiki/Pull-Request-Guidelines
Please reopen this after fixing suggestions.

- Check inline comments
- Add a test case, for reference check https://github.com/frappe/erpnext/blob/develop/erpnext/setup/doctype/company/test_company.py
- Add a screenshot of chart of accounts tree view

**Comment by nabinhait** [comment]
Also add the new chart in the dropdown field in Company and Setup Wizard. For reference, https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py#L113

**Comment by umairsy** [comment]
@khrlzhfr Thanks for the contribution. It's been long time these suggestions were proposed. Any chance you could pick it up? 

If you have good understanding on the iFRS standards, then we would like to work more closely with you for making ERPNext IFRS and GAAP compliant.

**Pull request closed without merging**